docs/source/services.md

```diff
@@ -1,70 +1,115 @@
-# JupyterHub services
+# Services
 
-JupyterHub 0.7 adds the notion of Services.
-A Service is a process that interacts with the Hub REST API.
-Services may perform actions such as shutting down user servers that have been idle for some time,
-or registering additional web servers that should also use the Hub's authentication
-and be served behind the Hub's proxy.
 
-There are two main characteristics of services:
+With version 0.7, JupyterHub adds support for 'Services'.
 
-1. Is it **managed** by JupyterHub?
-2. Does it have a web server that should be added to the proxy?
 
-If a `command` is specified for launching the service, it will be started and managed by the Hub.
-If a `url` is specified for where the service runs its own webserver,
-it will be added to the Hub's proxy at `/service/:service-name`.
+## JupyterHub's definition of a Service
 
-## Managed services
+When working with JupyterHub, a Service is defined as a process that interacts
+with the Hub's REST API. A Service may perform a specialized action or
+specific task. For example, each of these tasks can be a unique Service:
 
-**Managed** services are services that the Hub starts and is responsible for.
-These can only be local subprocesses of the Hub,
-and the Hub will take care of starting these processes and restarting them if they stop.
+- shutting down individuals' single user notebook servers that have been idle
+  for some time
+- registering additional web servers which should use the Hub's authentication
+  and be served behind the Hub's proxy.
 
-While there are similarities with notebook Spawners,
-there are no plans to support the same spawning abstractions as notebook.
-If you want to run these services in docker or other environments,
-you can register it as an external service below.
+Two main characteristics help define a Service:
 
-A managed service is characterized by the `command` specified for launching the service.
+1. Is the *Service* **managed** by JupyterHub?
+2. Does the *Service* have a web server that should be added to the proxy's 
+   table?
 
+Currently, a Service may be either a 'Managed Service' which is managed by
+JupyterHub, or an 'External Service' which runs its own web server and
+communicates operation instructions via the Hub's API.
+
+### Properties of a Service
+
+If a `command` is specified for launching the Service, the Service will be
+started and managed by the Hub.
+
+If a `url` is specified for where the Service runs its own web server,
+JupyterHub will add the Service to the Hub's proxy at 
+`/service/:service-name`.
+
+## Managed Service
```

Comment:
Should we change this to 'Hub-managed Service'?

As well as change 'External Service' to 'Externally-managed Service' or 'Self-managed Service'? Or simply leave as 'External Service'?